packages/msi/oh-my-posh.wxs

```diff
@@ -0,0 +1,39 @@
+<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
+	<Package Language="1033" Manufacturer="Jan De Dobbeleer" Name="Oh My Posh" UpgradeCode="b2a6bcaa-bda3-4c7f-8dec-d8665d7a9b69" Version="$(env.VERSION)" UpgradeStrategy="none">
+		<SummaryInformation Description="https://ohmyposh.dev" />
+		<MediaTemplate CabinetTemplate="tmpl{0}.cab" EmbedCab="yes" />
+		<SetProperty Id="ProgramFiles64Folder" Value="[LocalAppDataFolder]" Sequence="first" Before="CostFinalize" Condition="MSIINSTALLPERUSER=1" />
+		<StandardDirectory Id="ProgramFiles64Folder">
+			<Directory Id="ParentInstallDir" Name="oh-my-posh">
+				<Directory Id="INSTALLDIR" Name="bin" />
+				<Directory Id="THEMESDIR" Name="themes" />
+			</Directory>
+		</StandardDirectory>
+		<Feature Id="Installation" AllowAbsent="no" Description="https://ohmyposh.dev" Title="Oh My Posh">
+			<ComponentGroupRef Id="Files" />
+			<ComponentGroupRef Id="EnvironmentVariables" />
+		</Feature>
+	</Package>
+	<Fragment>
+		<ComponentGroup Id="Files">
+			<Component Id="Executable" Directory="INSTALLDIR">
+				<File Source="dist/oh-my-posh.exe" />
+			</Component>
+			<Files Directory="THEMESDIR" Include="../../themes/*.omp.*" />
+		</ComponentGroup>
+	</Fragment>
+	<Fragment>
+		<ComponentGroup Id="EnvironmentVariables">
+			<Component Id="SystemEnvironmentVariables" Condition="ALLUSERS=1" Directory="INSTALLDIR" Guid="ee903196-9032-4592-b461-25c96823bed7">
+				<Environment Id="SystemPath" Action="set" Name="Path" Part="first" Permanent="no" System="yes" Value="[INSTALLDIR]" />
+				<Environment Id="SystemThemesDir" Action="set" Name="POSH_THEMES_PATH" Part="all" Permanent="no" System="yes" Value="[THEMESDIR]" />
+				<Environment Id="SystemInstaller" Action="set" Name="POSH_INSTALLER" Part="all" Permanent="no" System="yes" Value="$(env.INSTALLER)" />
+			</Component>
+			<Component Id="UserEnvironmentVariables" Condition="MSIINSTALLPERUSER=1" Directory="INSTALLDIR" Guid="212dc904-b134-45b6-a647-a05ce417684a">
```

Comment:
Since `MSIINSTALLPERUSER=1` should be in the `Property` table by default (you'll need to add it), condition per-machine components on `ALLUSERS=1` and per-user components as `NOT ALLUSERS`. Windows Installer treats empty strings as unset so `NOT ALLUSERS` is safest. See <https://github.com/Azure/azure-dev/blob/6cd82ae9ff02e156a0056e994215191469442ab3/cli/installer/windows/azd.wxs#L78-L90> for an example.